### PR TITLE
More technologies

### PR DESCRIPTION
**Pull request by dasmb**

### CODE REVIEW

File: technologies/ammonite.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Ammonite"
+ring: "adopt"
+quadrant: "languages"
```
Comment (grahamar) [inline technologies/ammonite.md]: I'd say the language is Scala, so this is more of a tool or framework, no...?

File: technologies/awslambda.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Akka Lambda"
+ring: "trial"
+quadrant: "infrastructure"
+isNew: "false"
+---
+
+TODO: fill this out
```
Comment (grahamar) [inline technologies/awslambda.md]: Not sure what you mean by `Akka Lambda` can you add some context?

File: technologies/dev.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Flow dev"
```
Comment (grahamar) [inline technologies/dev.md]: I'd leave off internal proprietary tools like this, not sure there's value in tracking it... What do you think?

File: technologies/flowhackdaogenerator.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Flow Hack Dao Generator"
```
Comment (grahamar) [inline technologies/flowhackdaogenerator.md]: Ditto as above. I'd leave off internal proprietary tools like this, not sure there's value in tracking it... What do you think?

File: technologies/redis.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Redis"
```
Comment (grahamar) [inline technologies/redis.md]: Can we use "Redis (Elasticache)" and "Redis (Self-Hosted)", whichever you mean?

File: technologies/travisci.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Travis CI"
```
Comment (grahamar) [inline technologies/travisci.md]: This tech is already on the radar :)

File: technologies/flowjournaling.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Flow Journaling"
```
Comment (grahamar) [inline technologies/flowjournaling.md]: I'd either leave this one off the radar or change it to "Database Journalling"

File: technologies/ammonite.md (hunk 1 above)
Comment (dasmb) [reply technologies/ammonite.md]: Make the change if you like. Scala makes it so easy to create a DSL (that is also semantically correct Scala) I tend to consider uses of Scala with different semantic intents to be different languages. Think HOCON or SBT.

File: technologies/cloudwatch.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+---
+name: "AWS Cloudwatch"
+ring: "adopt"
+quadrant: "frameworks"
```
[the remark this answers is not in the document]
Comment (dasmb) [reply technologies/cloudwatch.md]: Operations.

File: technologies/awslambda.md (hunk 2 above)
Comment (dasmb) [reply technologies/awslambda.md]: Fat finger, meant AWS Lambda

File: technologies/dev.md (hunk 3 above)
Comment (dasmb) [reply technologies/dev.md]: I think it's very important to track proprietary tools that have behavior that mirrors tools in the marketplace (think about our discussions of `dev` vs. `kubectl` for managing deploys)

File: technologies/flowhackdaogenerator.md (hunk 4 above)
Comment (dasmb) [reply technologies/flowhackdaogenerator.md]: When we're pushing HDG as the code generator/schema generator, we're also rejecting tools like Slick or JOOQ that perform some of the same purpose. I think it's really important to track that.

File: technologies/ammonite.md (hunk 1 above)
Comment (dasmb) [reply technologies/ammonite.md]: E.g. we are using ammonite in place of Ruby or Python in areas where these tools are traditionally strong and Scala is not

**Pull request merged**